RxCocoa/iOS/UITableView+Rx.swift

```diff
@@ -110,7 +110,12 @@ extension Reactive where Base: UITableView {
      
      Example 
 
-        let dataSource = RxTableViewSectionedReloadDataSource<SectionModel<String, Double>>()
+        let dataSource = RxTableViewSectionedReloadDataSource<SectionModel<String, Double>>(
```

Comment:
Does `RxTableViewSectionedReloadDataSource` actually exist in this scenario? This isn't RxDataSources.

Reply:
@freak4pc It contained a comment for RxTableViewSectionedReloadDataSource. However, to ensure the purity of RxCocoa, it seems appropriate to remove the part of RxDataSources.

Reply:
I think we should entirely remove everything related to RxDataSources here - the code isn't part of this framework.

Reply:
Let's remove this example completely then.

Reply:
Somebody wants to remove this then?

Reply:
I'll take care of it, 2 minutes.